find_port.py

```diff
@@ -127,25 +124,16 @@ def main():
     if args.verbose:
         print('pyudev version = %s' % pyudev.__version__)
 
-    context = pyudev.Context()
 
     if args.list:
-        detected = False
-        for device in context.list_devices(subsystem='tty'):
-            if is_usb_serial(device, args):
-                print('USB Serial Device %s:%s%s found @%s\r' % (
-                      device['ID_VENDOR_ID'], device['ID_MODEL_ID'],
-                      extra_info(device), device.device_node))
-                detected = True
-        if not detected:
-            print('No USB Serial devices detected.\r')
-        return
```

Comment:
This return shouldn't have been removed. The way things are currently coded it does both the --list functionality and then the non --list functionality.